lib/util.js

```diff
@@ -5,7 +5,7 @@ const { resolve } = require('path');
 exports.glob = async function (dir) {
 	const output = [];
 	await totalist(dir, (name, abs) => output.push({ name, abs }));
```

Comment:
filter while pushing into collection & still allow typescript support
```suggestion
	await totalist(dir, (name, abs) => /\.[cm]?[tj]s$/.test(name) && output.push({ name, abs }));
```